RELEASE-NOTES.md

```diff
@@ -1,5 +1,21 @@
 # Prefect Release Notes
 
+## 2.0.1
+
+The response to Prefect 2 has been overwhelming in the best way possible. Thank you to the many community members who tried it out and gave us feedback! Thanks in particular to the students at this week's Prefect Associate Certification Course (PACC) in San Jose for their thoughtful recommendations. This release is a compilation of enhancements and fixes that make for a more resilient, performant, and refined Prefect experience.
+
+### Improvements and bug fixes
+- Schedules set via the API or UI are now preserved when building deployments from the CLI
+- JSON types are now coerced to none, following Javascript convention and supporting standards compatibility
+- The `prefect deployment execute` command has been removed
+- This repository now includes templates for pull requests and issues
+- A common request we immediately received was for unique deployment YAML names (to better anticipate multi-flow deployments) and an optional output flag to customize the name.
+- The `scheduler` and `flow-run-notifications` LoopServices have been made more resilient
+- Log inserts have been made more performant through smaller log batches
+- Local file system blocks can no longer be created from the UI
+- Support for unmapped values to Task.map has been added
```

Comment:
```suggestion
- Support for unmapped values to Task.map has been added as requested by Club42 members
```

Reply:
Feel free to skip this one but I thought this might be a cool note to show that we actually listen to their requests

Reply:
Definitely!